python/cudf/cudf/pylibcudf_tests/conftest.py

```diff
@@ -37,6 +37,36 @@ def numeric_pa_type(request):
     return request.param
 
 
+def _get_vals_of_type(pa_type, length):
+    """
+    Returns an list-like of random values of that type
+    """
+    if pa_type == pa.int64():
+        half = length // 2
+        negs = np.random.randint(-length, 0, half, dtype=np.int64)
```

Comment:
I think it's best practice to use a [generator](https://numpy.org/doc/stable/reference/random/generated/numpy.random.seed.html#numpy-random-seed) with a seed for reproducibility rather than using `np.randint`